packages/astro/src/core/messages.ts

```diff
@@ -91,58 +81,58 @@ export function serverStart({
 	}
 
 	const messages = [
-		`${emoji('🚀 ', '')}${bgGreen(black(` astro `))} ${green(`v${version}`)} ${dim(
-			`${isRestart ? 're' : ''}started in ${Math.round(startupTime)}ms`
-		)}`,
+		'',
+		`${bgGreen(bold(` astro `))} ${green(`v${version}`)} ${dim(`ready in`)} ${Math.round(
+			startupTime
+		)} ${dim('ms')}`,
 		'',
 		...localUrlMessages,
 		...networkUrlMessages,
 		'',
 	];
-	return messages
-		.filter((msg) => typeof msg === 'string')
-		.map((msg) => `  ${msg}`)
-		.join('\n');
+	return messages.filter((msg) => typeof msg === 'string').join('\n');
 }
 
-export function telemetryNotice(packageManager = 'npm') {
-	const headline = `${cyan('◆')} Astro collects completely anonymous usage data.`;
-	const why = dim('  This optional program helps shape our roadmap.');
-	const disable = dim(`  Run \`${packageManager} run astro telemetry disable\` to opt-out.`);
-	const details = `  Details: ${underline('https://astro.build/telemetry')}`;
-	return [headline, why, disable, details].map((v) => '  ' + v).join('\n');
+export function telemetryNotice() {
+	const headline = blue(`▶ Astro collects anonymous usage data.`);
+	const why = '  This information helps us improve Astro.';
+	const disable = `  Run "astro telemetry disable" to opt-out.`;
+	const details = `  ${cyan(underline('https://astro.build/telemetry'))}`;
+	return [headline, why, disable, details].join('\n');
 }
 
 export function telemetryEnabled() {
-	return `${green('◉')} Anonymous telemetry is now ${bgGreen(black(' enabled '))}\n  ${dim(
-		'Thank you for improving Astro!'
-	)}\n`;
+	return [
+		green('▶ Anonymous telemetry ') + bgGreen(' enabled '),
+		`  Thank you for helping us improve Astro!`,
+		``,
+	].join('\n');
 }
 
 export function telemetryDisabled() {
-	return `${yellow('◯')} Anonymous telemetry is now ${bgYellow(black(' disabled '))}\n  ${dim(
-		"We won't ever record your usage data."
-	)}\n`;
+	return [
+		green('▶ Anonymous telemetry ') + bgGreen(' disabled '),
+		`  Astro is no longer collecting anonymous usage data.`,
+		``,
+	].join('\n');
 }
 
 export function telemetryReset() {
-	return `${cyan('◆')} Anonymous telemetry has been ${bgCyan(black(' reset '))}\n  ${dim(
-		'You may be prompted again.'
-	)}\n`;
+	return [green('▶ Anonymous telemetry preferences reset.'), ``].join('\n');
 }
 
 export function fsStrictWarning() {
-	return yellow(
-		'⚠️ Serving with vite.server.fs.strict: false. Note that all files on your machine will be accessible to anyone on your network!'
-	);
+	const title = yellow('▶ ' + `${bold('vite.server.fs.strict')} has been disabled!`);
+	const subtitle = `  Files on your machine are likely accessible on your network.`;
```

Comment:
I just felt like THIS COULD HAPPEN is more interpreted as a warning phrase, even if, it totally does happen. The original seemed like a pretty strong warning, and the new doesn't feel like it would jump out to the reader as much. Just a thought!